src/test/java/bio/terra/common/GoogleResourceUtils.java

```diff
@@ -0,0 +1,39 @@
+package bio.terra.common;
+
+import bio.terra.app.model.GoogleRegion;
+import bio.terra.buffer.model.ResourceInfo;
+import bio.terra.model.BillingProfileModel;
+import bio.terra.service.resourcemanagement.BufferService;
+import bio.terra.service.resourcemanagement.google.GoogleProjectResource;
+import bio.terra.service.resourcemanagement.google.GoogleProjectService;
+import com.google.api.client.util.Lists;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GoogleResourceUtils {
+  @Autowired private BufferService bufferService;
+  @Autowired private GoogleProjectService projectService;
+
+  public GoogleProjectResource buildProjectResource(BillingProfileModel profile) throws Exception {
+    String role = "roles/bigquery.jobUser";
+    String stewardsGroupEmail = "group:JadeStewards-dev@dev.test.firecloud.org";
+    List<String> stewardsGroupEmailList = Lists.newArrayList();
+    stewardsGroupEmailList.add(stewardsGroupEmail);
+    Map<String, List<String>> roleToStewardMap = new HashMap<>();
+    roleToStewardMap.put(role, stewardsGroupEmailList);
```

Comment:
I know this is copied code, but I think it can be improved. I would use the JDK collection constructors here. Also I don't think it's necessary to declare variables for each element since they're only used once here.
```suggestion
    var roleToStewardMap = Map.of("roles/bigquery.jobUser", List.of("group:JadeStewards-dev@dev.test.firecloud.org""));
```